integration-libs/cdc/organization/administration/cdc-administration.module.ts

```diff
@@ -0,0 +1,55 @@
+/*
+ * SPDX-FileCopyrightText: 2023 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { NgModule } from '@angular/core';
+
+import { CmsConfig, provideDefaultConfig } from '@spartacus/core';
+import {
+  unitsCmsConfig,
+  userCmsConfig,
+  UserListService,
+} from '@spartacus/organization/administration/components';
+import { B2BUserService } from '@spartacus/organization/administration/core';
+import { CdcB2BUserService } from './cdc-b2b-user.service';
+import { CdcUserListService } from './cdc-user-list.service';
+
+@NgModule({
+  providers: [
+    //to override UserListService in ListComponent
+    provideDefaultConfig(<CmsConfig>{
+      cmsComponents: {
+        ManageUsersListComponent: {
+          providers: [
+            {
+              provide: UserListService,
+              useExisting: CdcUserListService,
+            },
+            userCmsConfig.cmsComponents?.ManageUsersListComponent?.providers ||
+              [],
+          ],
+        },
+      },
+    }),
+    //to override B2BUserService in UserDetailsComponent, UnitUserListComponent
+    { provide: B2BUserService, useClass: CdcB2BUserService },
+    //to override B2BUserService in UnitUserRolesCellComponent
+    provideDefaultConfig(<CmsConfig>{
+      cmsComponents: {
+        ManageUnitsListComponent: {
+          providers: [
+            {
+              provide: B2BUserService,
+              useExisting: CdcB2BUserService,
+            },
+            unitsCmsConfig.cmsComponents?.ManageUnitsListComponent?.providers ||
+              [],
+          ],
+        },
+      },
+    }),
```

Comment:
Is this all _really_ neeed? Can't we replace it with just 2 lines providers in `@NgModule.providers`:
```ts
{ provide: UserListService, useExisting: CdcUserListService },
{ provide: B2BUserService, useExisting: CdcB2BUserService },
```
?

---
Reason:
I can see `UserListService` is already `providedIn: root`: https://github.com/SAP/spartacus/blob/bd5c7c93a2aa46f30600ee3d3fd09835beebe6bf/feature-libs/organization/administration/components/user/services/user-list.service.ts#L37-L40

So I guess we don't need to over-provide it on the cms-mapping level, but it should suffice to over-provide it just on the level of `@NgModule.

Thanks to that, we could avoid potentially unnecessary referencing `userCmsConfig.cmsComponents?.ManageUsersListComponent?.providers` and merging it with our custom over-provider on the cms-mapping level.

and Analogically for `B2BUserService`.

Reply:
Hi Kris, making the suggested change is not overriding B2BUserService with CdcB2BUserService in UnitUserRolesCellComponent and UserListService with CdcUserListService in ListComponent.